tests_pytest/pytorch_tests/integration_tests/core/quantization/test_set_node_quantization_config.py

```diff
@@ -0,0 +1,230 @@
+
+    def get_tpc(self):
+        base_cfg, mx_cfg_list, default_config = self.get_op_qco()
+        tpc = self.generate_tpc_local(default_config, base_cfg, mx_cfg_list)
+        return tpc
+
+    def representative_data_gen(self, shape=(3, 8, 8), num_inputs=1, batch_size=2, num_iter=1):
+        for _ in range(num_iter):
+            yield [torch.randn(batch_size, *shape)] * num_inputs
+
+    def get_float_model(self):
+        class BaseModel(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = torch.nn.Conv2d(in_channels=3, out_channels=3, kernel_size=3)
+                self.conv2 = torch.nn.Conv2d(in_channels=3, out_channels=3, kernel_size=3)
+                self.relu = torch.nn.ReLU()
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x = self.conv2(x)
+                x = torch.add(x, 2)
+                x = self.relu(x)
+                return x
+        return BaseModel()
+
+    def get_test_graph(self, core_config):
+        float_model = self.get_float_model()
+        fw_info = DEFAULT_PYTORCH_INFO
+
+        fw_impl = PytorchImplementation()
+        graph = fw_impl.model_reader(float_model,
+                                     self.representative_data_gen)
+        graph.set_fw_info(fw_info)
+
+        tpc = self.get_tpc()
+        attach2pytorch = AttachTpcToPytorch()
+        fqc = attach2pytorch.attach(
+            tpc, core_config.quantization_config.custom_tpc_opset_to_layer)
+        graph.set_fqc(fqc)
+
+        return graph
+
+    # test case for set_manual_activation_bit_width
+    """
+    Test Items Policy:
+        - How to specify the target layer: Options(type/name)
+        - Target attribute information: Options(kernel) 
+        - Bit width variations: Options(2, 4, 16)
+    """
+    test_input_1 = (NodeNameFilter("conv1"), 2, PYTORCH_KERNEL)
+    test_input_2 = (NodeTypeFilter(nn.Conv2d), 16, PYTORCH_KERNEL)
+    test_input_3 = ([NodeNameFilter("conv1"), NodeNameFilter("conv2")], [4, 8], [PYTORCH_KERNEL, PYTORCH_KERNEL])
+
+    test_expected_1 = ({"conv1": {PYTORCH_KERNEL: 2}, "conv2": {PYTORCH_KERNEL: 8}})
+    test_expected_2 = ({"conv1": {PYTORCH_KERNEL: 16}, "conv2": {PYTORCH_KERNEL: 16}})
+    test_expected_3 = ({"conv1": {PYTORCH_KERNEL: 4}, "conv2": {PYTORCH_KERNEL: 8}})
+
+    @pytest.mark.parametrize(
+        ("inputs", "expected"), [
+        (test_input_1, test_expected_1),
+        (test_input_2, test_expected_2),
+        (test_input_3, test_expected_3),
+    ])
+    def test_manual_weights_bitwidth_selection(self, inputs, expected):
+        core_config = CoreConfig()
+        graph = self.get_test_graph(core_config)
+
+        core_config.bit_width_config.set_manual_weights_bit_width(inputs[0], inputs[1], inputs[2])
+
+        updated_graph = set_quantization_configuration_to_graph(
+            graph, core_config.quantization_config, core_config.bit_width_config,
+            False, False
```

Comment:
the last two arguments (False, False) have default values in the function so you are not obligated to pass them here. If you want to pass them two distinguish between two different test case (for example - with mixed precision or without) then add a key-argument name to the function call (for clarity).

Also, I think you should add a test where mixed precision is enabled - this is a use case that is important to verify with manual bit-width selection (if a node has mixed precision candidates, but also set with manual bit width - the manual request should get priority)